### PR TITLE
reg_district not found

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

File: js_sandbox/test/test-go-nike-ghr.js [hunk 1]
```diff
@@ -293,6 +295,31 @@ describe("When using the USSD line", function() {
...
+                next_state: "reg_thanks",
+                response: (
+                    "^Welcome Ni Nyampinga club member! We want to know you better. " +
+                    "For each set of 4 questions you answer, you enter a lucky draw to " +
+                    "win 100 RwF weekly.[^]" +
+                    "1. Continue$"
+                )
+            });
+                p.then(done, done);
+            });
```
Comment (smn) [inline js_sandbox/test/test-go-nike-ghr.js]: to the left

File: js_sandbox/test/test-go-nike-ghr.js [hunk 2]
```diff
@@ -293,6 +295,31 @@ describe("When using the USSD line", function() {
                 p.then(done, done);
             });
 
+        it("entering a duplicate sector (like live) should ask for district", function (done) {
+            var user = {
+                current_state: 'reg_district',
```
Comment (smn) [inline js_sandbox/test/test-go-nike-ghr.js:300]: I'm confused how someone can end up in this state to begin with. Can you explain that a bit?

**Comment by smn** [comment]
Some context as to what you're fixing here would be helpful and why is py26 failing on travis with an importlib error?

**Comment by imsickofmaps** [comment]
Sorry - I was thinking you'd have seen the JIRA ticket - bad assumption with your work volume.

File: js_sandbox/test/test-go-nike-ghr.js (hunk 2 above)
Comment (imsickofmaps) [inline js_sandbox/test/test-go-nike-ghr.js:300]: If you enter a sector that isn't unique we ask for district too. So the thanks and end state builder returns that instead. But when returning the reg_district state doesn't exist. So I added it

**Comment by imsickofmaps** [comment]
I've not changed anything related to the python part of this project. I'll see if It's just travis being travis.

File: js_sandbox/test/test-go-nike-ghr.js (hunk 2 above)
Comment (smn) [inline js_sandbox/test/test-go-nike-ghr.js:300]: ok, the fact that you need to define the state twice probably hints at something that needs to be improved but I'll leave that for now. Perhaps add a note explaining the issue with the duplicate state definition?

**Comment by smn** [comment]
:+1: but making sure the python builds pass on Travis would be ++

**Pull request merged**